dlt/destinations/job_impl.py

```diff
@@ -79,7 +79,8 @@ def job_id(self) -> str:
 
 class ReferenceFollowupJobRequest(FollowupJobRequestImpl):
     def __init__(self, original_file_name: str, remote_paths: List[str]) -> None:
-        file_name = os.path.splitext(original_file_name)[0] + "." + "reference"
+        job_info = ParsedLoadJobFileName.parse(original_file_name)
```

Comment:
maybe ParsedLoadJobFileName could have a method `to_reference_file_name` that does this? And you can use it here?